src/Sentry/Extensibility/RequestBodyExtractionDispatcher.cs

```diff
@@ -31,22 +31,17 @@ public RequestBodyExtractionDispatcher(IEnumerable<IRequestPayloadExtractor> ext
         /// </summary>
         /// <param name="request">The request.</param>
         /// <returns>A serializable representation of the payload.</returns>
-        public object ExtractPayload(IHttpRequest request)
+        public object? ExtractPayload(IHttpRequest? request)
```

Comment:
The return or the parameter? Both seem to be able to be null in some scenarios.

Reply:
In `BaseRequestPayloadExtractor` for example

Reply:
Sorry I wasn't clearn. I mean the IHttpRequest. I rather the caller not invoke this if no request object exists since there's no sense in running 'extract payload' in that case